CHANGELOG.md

```diff
@@ -3,13 +3,16 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+### Breaking Change
+- `Floe::Workflow::Runner.docker_runner=` is now `Floe.set_runner` ([#152](https://github.com/ManageIQ/floe/pull/152))
```

Comment:
Could we add an alias so we don't have to do a major version bump, just deprecate the old one and drop it when we do a major bump?